tests/utils/inputparams/testparams.go

```diff
@@ -42,10 +43,10 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
+	return volName + "_volume_" + strconv.FormatInt(rand.Int63(), 10)
```

Comment:
We have never run into this problem. I guess this might be your environment issue. Can you try to write a small go program to verify your environment?